mountpoint-s3-crt-sys/UPDATING_CRT.md

```diff
@@ -0,0 +1,35 @@
+# CRT Submodules
+
+Mountpoint for Amazon S3 imports the required [AWS Common Runtime (CRT)](https://docs.aws.amazon.com/sdkref/latest/guide/common-runtime.html) libraries through git submodules, located under the `mountpoint-s3-crt-sys/crt` directory.
+
+## Update Submodules
+
+The CRT submodules can be updated by following these steps:
+
+1. Update every submodule to the latest tagged release. E.g. by running:
+
+   ```sh
+   git submodule foreach 'git fetch -q --tags && git checkout --recurse-submodules `git tag -l --sort=-v:refname | head -1`'
+   ```
+
+2. Review the commit history, in particular for `aws-c-s3`, looking for changes that may affect `mountpoint-s3` (bug fixes, API changes, etc.). E.g.:
+
+   ```sh
+   git diff --submodule mountpoint-s3-crt-sys/crt/aws-c-s3 
+   ```
+
+3. Build and test `mountpoint-s3`.
+
+4. Stage and commit the changes:
+
+   ```sh
+   git commit -a -s -m "Update CRT submodules to latest releases"
+   ```
```

Comment:
Unless obvious, I'd generally recommend to use the full flag names in documentation.

For instance, something like:

   ```sh
   git commit --all --signoff \
       -m "Update CRT submodules to latest releases"
   ```

I'm also a bit wary of recommending the use of `--all` flag, but fine for this.

Not blocking this PR on any of this.